lite/tests/segmentation/test_stability.py

```diff
@@ -0,0 +1,85 @@
+from random import choice
```

Comment:
we can no longer run all `lite` tests at once (using `pytest lite/tests/*`) since these test files share the same names as the test files in `lite/tests/classification/*`. can you update these file names to be unique?
<img width="568" alt="image" src="https://github.com/user-attachments/assets/96b10801-bc6c-4828-9420-7f7daad18f26">

Reply:
as a user, having to run all of the tests separately before committing is a major pain. you may be able to [find a workaround here](https://docs.pytest.org/en/stable/explanation/goodpractices.html#choosing-an-import-mode) if you want to keep the names the exact same, but I don't think this is a pytest best practice.

> This allows the tracking of coverage to be separate

you'll still be able to track coverage separately since you're testing individual directories

> finding equivalent tests in another task type easier

you can append "_clf", "_det", "_seg" to the file names if you want. I don't see this benefit outweighing the annoyance of having to run all of the tests separately.

Reply:
Adding the init files got `pytest lite/tests` working again. I had removed these b/c I thought they werent necessary.